src/scaffold/collection/coordinator.rs

```diff
@@ -231,6 +231,22 @@ fn add_delete_link_in_delete_function(
             ));
         }
         CollectionType::ByAuthor => {
+            delete_link_stmts.insert(
+                0,
+                r#"
+                let record = match details {{
+                    Details::Record(details) => Ok(details.record),
+                    _ => Err(wasm_error!(WasmErrorInner::Guest(String::from(
+                        "Malformed get details response"
+                    )))),
+                }}?;
+            "#
+                .to_string(),
+            );
+            delete_link_stmts.insert(0, format!(r#"
```

Comment:
Yes, `insert` will add the string to the vec at index `0` in this case, then shift the other elements to the right, basically prepending to the vec, which is what we want here instead of calling `push` which appends to the vec